dtos/movies.go

```diff
@@ -0,0 +1,13 @@
+package dtos
+
+import (
+	"github.com/ashwinspg/explore-golang/utils"
+)
+
+//Movie - DTO
+type Movie struct {
+	UUID string            `json:"uuid"`
+	Info utils.PropertyMap `json:"info"`
```

Comment:
Info can just be the object that Moviebuff returns as well

Reply:
you can see the commit [here](https://github.com/ashwinspg/explore-golang/pull/3/commits/5e4b213efb2ca36fe9d3fdc79af960849cfc00ff)